playbooks/example-source/playbook.yml

```
@@ -9,7 +9,7 @@
         msg: >
           "You must update Ansible to at least 2.8 to use this version of Pulp 3 Installer."
   roles:
-    - pulp_all_services
-    - pulp_devel
+    - pulp.pulp_installer.pulp_all_services
+    - pulp.pulp_installer.pulp_devel
```

I learned that instead of doing fqdn, we can do:
```suggestion
collections:
    - pulp.pulp_installer
roles:
    - pulp_all_services
    - pulp_devel
```
I did it in a hack-ish way here:
https://github.com/pulp/pulp_installer/runs/2175295306?check_suite_focus=true
https://github.com/pulp/pulp_installer/pull/562